stubs/pytest-snapshot/pytest_snapshot/plugin.pyi

```diff
@@ -0,0 +1,27 @@
+import re
+from collections.abc import Generator
+from pathlib import Path
+from types import TracebackType
+from typing import Any
+
+import _pytest.config.argparsing
+import pytest
+
+PARAMETRIZED_TEST_REGEX: re.Pattern[str]
+
+def pytest_addoption(parser: _pytest.config.argparsing.Parser) -> None: ...
+@pytest.fixture
+def snapshot(request: pytest.FixtureRequest) -> Generator[Snapshot, Any, None]: ...
```

Comment:
We usually use `Generator[Snapshot]` (although many annotations haven't been updated since type var defaults have been introduced), which is equivalent to `Generator[Snapshot, None, None]`. If it's important to you that anything can be sent into the generator, use `object` instead of `Any`, which we used for ignored parameters.